packages/cli-exec/README.md

```diff
@@ -5,89 +5,110 @@ Percy CLI commands for running a local snapshot server using [`@percy/core`](./p
 ## Commands
 <!-- commands -->
 * [`percy exec`](#percy-exec)
-* [`percy exec:ping`](#percy-execping)
 * [`percy exec:start`](#percy-execstart)
 * [`percy exec:stop`](#percy-execstop)
+* [`percy exec:ping`](#percy-execping)
 
-## `percy exec`
+### `percy exec`
 
 Start and stop Percy around a supplied command
 
 ```
-USAGE
-  $ percy exec
-
-OPTIONS
-  -P, --port=port                                  [default: 5338] server port
-  -c, --config=config                              configuration file path
-  -d, --dry-run                                    print logs only, do not run asset discovery or upload snapshots
-  -h, --allowed-hostname=allowed-hostname          allowed hostnames to capture in asset discovery
-  -q, --quiet                                      log errors only
-  -t, --network-idle-timeout=network-idle-timeout  asset discovery network idle timeout
-  -v, --verbose                                    log everything
-  --debug                                          debug asset discovery and do not upload snapshots
-  --disable-cache                                  disable asset discovery caches
-  --parallel                                       marks the build as one of many parallel builds
-  --partial                                        marks the build as a partial build
-  --silent                                         log nothing
-
-EXAMPLES
+Usage:
+  $ percy exec [options] -- <command>
```

Comment:
Lowkey the biggest win from this refactor is being able to clearly show examples like this.